edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -206,8 +208,21 @@ impl MakeModuleRuntime for DockerModuleRuntime {
         //      https://github.com/rust-lang/rust-clippy/issues/3730
         #[allow(clippy::result_map_unwrap_or_else)]
         let created = init_client(settings.moby_runtime().uri())
-            .map(|client| {
+            .and_then(|client| {
+                let home_dir = settings.homedir();
```

Comment:
what if home_dir is empty?

Reply:
home_dir is configured via config.yaml and it is default var/lib/iotedge. I was searching for existing tests to see if there is a check for home_dir to be empty. but there is not one today. I will add the checks